### ISSUE TITLE
[Bug]: Can access and login only on localhost but login from another machine login generate an error

### ISSUE THREAD

**Issue by eliassal** [featbit/featbit#150]
### Contact Details

salam@salam.hd.free.fr

### What happened?

I ran the docker compose all containers up and running and I was able to log on the machine using http://mysevername:8081.
When I try same thing but from another machine, I get the login page but when I enter login and password, it indicates has happened and get stuck on the login page

### To Reproduce

1. On the local machine visit http://myhostname:8081 (not localhost)
2. Login page is displayed
3. after entering credentials I get into the site
4. go to another machine in the same LAN, use the same URL and credentials
5. click on login
6. error happened and screen gets stiuck on the login page I used 2 different browsers, firefox as well as Edge

### Screenshots

on local machine where containers run

![image](https://user-images.githubusercontent.com/8447864/206253742-010e2ff6-1a24-4879-9fdb-9459efb13ca7.png)

from remote machine

![image](https://user-images.githubusercontent.com/8447864/206253348-4f73ceed-5ae0-4d41-ba4a-a3cd6473b0ed.png)

### Browsers

Firefox Edge

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cosmos-explorer** [comment]
Hello @eliassal, thank you for creating an issue. 

The problem comes from the url of the API server, by default, the value is **http://localhost:5000**, this address is only accessible from the same machine, it needs to be replaced by the **IP address** or **domain name** of the machine on which the API server is running, http://187.25.69.897:5000 or http://www.mydomain.com:5000 for example, **the address and port must be accessible from the second machine**。

You can modifiy the parameter in the **docker-compose.yml** file (the file is located in the root directory of the repo) as follows:
![image](https://user-images.githubusercontent.com/88151306/206261985-a0d93c33-5039-4057-8e88-8ea05f344c9f.png)

For a better understanding of the architecture, please read this doc https://featbit.gitbook.io/docs/tech-stack/architecture,
the overall architecture is like this picture:
![image](https://user-images.githubusercontent.com/88151306/206262204-7cc9143d-f571-4421-88c5-ba65ce7cb7de.png)

Please reach to me if you have other questions, or join our slack channel.

**Comment by eliassal** [comment]
Yes, you are correct, it is working like a charm  from remote machine. So mqny thanks for your prompt response

**Comment by cosmos-explorer** [comment]
> Yes, you are correct, it is working like a charm from remote machine. So mqny thanks for your prompt response

No problem, I'm closing the issue.

**Comment by eliassal** [comment]
@cosmos-explorer I started reading the archi and the doc, but I have a very quick question. My use case is I have a project in Azure Devops server on prem, not online, source code managed by git repo. How do I link featBit to the project knowing deployment happens inside a build or release pipeline? Not very clear to me how to establish this link.
Thanks for your help

**Comment by deleteLater** [comment]
Hello @eliassal, thanks for contacting us.

> How do I link featBit to the project knowing deployment happens inside a build or release pipeline? 

What does this mean? I'm not quite sure what you're trying to do, can you explain it further?

**Comment by cosmos-explorer** [comment]
@eliassal I might understand your question. Do you mean you want to deploy FeatBit by using Azure Devops pipeline? If so, I think you could do something as what we are doing. I never used Azure Devops Server, but I suppose it is similar with Azure Devops sevice (the cloud version).

1. Create a screat in Key Vault with the ip or domain name as the value
2. Add a step in your pipleline to replace the value of API_URL in docker-compose.ymal
3. Do whatever you have done before

Please me tell if those suggestions help.

**Comment by eliassal** [comment]
Yes exact, this is what I want to achieve and start testing your interesting platform as II already use Azure Devops online and on-prem. FeatureFlag system 
In azure devops we can create tokens as well as using user/password for interacting on different levels, even we can call Azure Devops APIs  and act according to outcome. If I understand well your suggestion, I need to create a token its name the IP of my on-prem server (not sure if it should be the url/IP only or the full url for the project?) then modify on your platform the url in docker-compose.
Let me here ask the following question : If in case the url will be for a project, in this case I can only work with 1 single project, no?

**Comment by cosmos-explorer** [comment]
@eliassal 
First of all, the value of the token should be the IP or url, with the port 5000 of course, as long as the IP or url is accessible from the internet 

What do you mean by project? 

As for FeatBit, you can create as many projects (Web, Mobile, Server etc.) as you want, and each project can have as many environments (Dev, Staging, Prod etc.) as needed. Each environment has a secret and you can use that secret to connect a SDK to FeatBit.

**Comment by eliassal** [comment]
OK, let me go through the tutorial/doc and come back to you later

**Comment by cosmic-flood** [comment]
> salam@salam.hd.free.fr

Hi, 

How are things going on your end?
Don't hesitate to ask if you encounter any problem. 

You can create issue,[ join our slack](https://join.slack.com/t/featbit/shared_invite/zt-1ew5e2vbb-x6Apan1xZOaYMnFzqZkGNQ) or send me emails (mailto:hu-beau@outlook.com)

:)
